canonical-webteam/task-creator
# Snap the app

# Summary

Create a snap from package

# QA

- `snapcraft cleanbuild`
- ` snap install task_amd64.snap  --devmode --dangerous`
- `task 395 3 "toto"`
- Should start asking the questions
